live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/content/IngestContentCommand.java

```diff
@@ -23,15 +23,10 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  * #L%
  */
-
 package org.alfresco.hxi_connector.live_ingester.domain.usecase.content;
 
-import static org.alfresco.hxi_connector.live_ingester.domain.utils.EnsureUtils.ensureNotBlank;
-
-public record IngestContentCommand(String nodeId)
-{
-    public IngestContentCommand
-    {
-        ensureNotBlank(nodeId, "Node id cannot be blank");
```

Comment:
Can the node id be null for error cases?

Reply:
idk maybe - it is fail-fast concreted approach - but I didn't deleted this check just renamed "IngestContentCommand" to "TriggerContentIngestionCommand" and created new "IngestContentCommand" and git didn't handle it correctly

Reply:
Weird! Sounds more like an IntelliJ bug than a git bug though.